github/repos_pages.go

```diff
@@ -36,6 +44,46 @@ type PagesBuild struct {
 	UpdatedAt *Timestamp  `json:"updated_at,omitempty"`
 }
 
+// EnablePageSite allows to enable GitHub Pages
```

Comment:
How about:

```go
// EnablePageSite enables GitHub Pages for the named repo.
```

Please always make the comments complete sentences to keep the auto-generated Godocs of high quality and easy to read.